plugins/woocommerce-blocks/assets/js/blocks/product-collection/collections/hand-picked.tsx

```diff
@@ -0,0 +1,102 @@
+/**
+ * External dependencies
+ */
+import type { InnerBlockTemplate } from '@wordpress/blocks';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { INNER_BLOCKS_PRODUCT_TEMPLATE } from '../constants';
+import {
+	CoreCollectionNames,
+	CoreFilterNames,
+	ETimeFrameOperator,
+} from '../types';
+
+const Icon = () => {
+	return (
+		<svg
```

Comment:
The SVG here breaks the syntax highlight in GH. I'm not sure how to handle SVGs in this project since this seems to be the first one that is not available from the [@wordpress/icons package](https://wordpress.github.io/gutenberg/?path=/story/icons-icon--library):

<img width="259" alt="Screenshot 2024-09-06 at 13 18 30" src="https://github.com/user-attachments/assets/ff6aed91-f6c3-4180-8878-a84d228d438f">


Any ideas, @imanish003 @kmanijak @xristos3490?

Reply:
Maybe we can use the same approach as in the [Breadcrumbs block](https://github.com/woocommerce/woocommerce/blob/3d5e7c3610b543db31f4a3adca39597a49a8e4e7/plugins/woocommerce-blocks/assets/js/blocks/breadcrumbs/index.tsx#L31-L38)? 
- Separate file for the icon: I don't think it's necessary here. If we will create a new file for icon then probably we will need to create a new folder for icons in Product Collection block.
- `Icon` from `@wordpress/icons` package: I believe we can use it here as well. Here is how it looks like:
```tsx
import { Icon } from '@wordpress/icons';

<Icon
    icon={ handPickedCollectionIcon }
/>
```

What do you think?

Reply:
> Maybe we can use the same approach as in the [Breadcrumbs block](https://github.com/woocommerce/woocommerce/blob/3d5e7c3610b543db31f4a3adca39597a49a8e4e7/plugins/woocommerce-blocks/assets/js/blocks/breadcrumbs/index.tsx#L31-L38)?

Oh that's perfect - missed this one! Thanks! 🙇

Reply:
Updated in de596e3e1197b7a8075598383c33c5d07cb32f2b

Reply:
For some reason, the GH syntax highlighting for this file is still broken. 🙄 🤷

Reply:
Haha IMO As long as the functionality is correct, we're good to go 🤔